packages/kernel-5.15/latest-kernel-full-config.sh

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env bash
```

Comment:
Somewhere in other PR there was a mention of not using `.sh` at the end of the scripts. Let's stick to that convention.

Reply:
I can't find the PR where it was asked to not use `.sh`. I'll back off from it now, but we should standardize the usage of `.sh` in our scripts because we have some with `.sh` and some without it.